.github/workflows/image-builder.yaml

```diff
@@ -0,0 +1,38 @@
+name: image-builder
+
+on:
+  schedule:
+    - cron: "0 6 */14 * *"
+jobs:
+  image-builder:
+    runs-on: [self-hosted, linux]
+    steps:
+    - name: Checkout image-builder
+      uses: actions/checkout@v3
+      with:
+        repository: 'outscale-vbr/image-builder'
+        path: "add-outscale-image"
+        ref: add-outscale-image
+    - name: Use latest Packer
+      uses: hashicorp-contrib/setup-packer@v1
+      with:
+        packer-version: 1.7.10
+    - name: Launch image-builder
+      run: |
+        sudo pip3 install setuptools-rust
+        sudo pip3 install --upgrade pip
+        export PATH=$HOME/.local/bin:$GITHUB_WORKSPACE/image-builder/images/capi/.local/bin:$PATH
+        make deps-osc
+        groupadd -r packer && useradd -m -s /bin/bash -r -g packer packer
+        chown -R packer:packer ${{ github.workspace }}/add-outscale-image
+        runuser -l packer -c "export LANG=C.UTF-8; export LC_ALL=C.UTF-8; export PACKER_LOG=1; export PATH=~packer/.local/bin/:$GITHUB_WORKSPACE/image-builder/images/capi/.local/bin:$PATH; export OUTSCALE_ACCESSKEYID=${OUTSCALE_ACCESSKEYID}; export OUTSCALE_SECRETKEYID=${OUTSCALE_SECRETKEYID}; export OUTSCALE_REGION=${OUTSCALE_REGION}; cd $GITHUB_WORKSPACE/add-outscale-image/images/capi;  ./scripts/ci-outscale-nightly.sh"
+      shell: bash
+      working-directory: "${{ github.workspace }}/add-outscale-image/images/capi"
+      env:
+         OUTSCALE_ACCESSKEYID: ${{secrets.OSC_ACCESS_KEY}}
+         OUTSCALE_SECRETKEYID: ${{secrets.OSC_SECRET_KEY}}   
+         OUTSCALE_REGION: ${{secrets.OSC_REGION}}
+         CRYPTOGRAPHY_DONT_BUILD_RUST: 1
+         PACKER_LOG: 1 
+         LANG: C.UTF-8
+         LC_ALL: C.UTF-8
```

Comment:
Exaplin why you must set these ?

Reply:
https://github.com/pypa/pip/issues/10219